ObjectiveGit/GTIndex.m

```diff
@@ -308,17 +316,17 @@ - (BOOL)enumerateConflictedFilesWithError:(NSError **)error usingBlock:(void (^)
 			return NO;
 		}
 
-		GTIndexEntry *blockAncestor;
+		GTIndexEntry *blockAncestor = nil;
 		if (ancestor != NULL) {
 			blockAncestor = [[GTIndexEntry alloc] initWithGitIndexEntry:ancestor];
 		}
 
-		GTIndexEntry *blockOurs;
+		GTIndexEntry *blockOurs = nil;
```

Comment:
I actually don't care too much about this, but in the [ARC transitioning release notes](https://developer.apple.com/library/content/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html) it reads "Using ARC, strong, weak, and autoreleasing stack variables are now implicitly initialized with nil.".